mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -358,35 +357,36 @@ class ExperimentView extends Component {
     metricRanges,
     selected) {
 
+    let columnKey = 0;
     const numParams = paramKeyList.length;
     const numMetrics = metricKeyList.length;
     const row = [
-      <td><input type="checkbox" checked={selected}
+      <td key={columnKey++}><input type="checkbox" checked={selected}
         onClick={() => onCheckbox(runInfo.run_uuid)}/></td>,
-      <td>
+      <td key={columnKey++}>
         <Link to={Routes.getRunPageRoute(runInfo.experiment_id, runInfo.run_uuid)}>
           {runInfo.start_time ? Utils.formatTimestamp(runInfo.start_time) : '(unknown)'}
         </Link>
       </td>,
-      <td>{Utils.formatUser(runInfo.user_id)}</td>,
-      <td>{Utils.renderSource(runInfo)}</td>,
-      <td>{Utils.renderVersion(runInfo)}</td>,
+      <td key={columnKey++}>{Utils.formatUser(runInfo.user_id)}</td>,
+      <td key={columnKey++}>{Utils.renderSource(runInfo)}</td>,
+      <td key={columnKey++}>{Utils.renderVersion(runInfo)}</td>,
     ];
 
     let firstParam = true;
     paramKeyList.forEach((paramKey) => {
       const className = firstParam ? "left-border": undefined;
       firstParam = false;
       if (paramsMap[paramKey]) {
-        row.push(<td className={className}>
+        row.push(<td className={className} key={columnKey++}>
```

Comment:
These numeric keys may mess things up if people change the filters and hide some of the columns later, because React will think that the old columns have the correct key. If keys are required here, they should be based on the parameter and value names, so that React keeps the element only if the same param/metric is supposed to stay in view.